Automattic/sensei
# Add Psalm to Sensei

Resolves part of #7094 

## Proposed Changes
* Add Psalm to Sensei

## Testing Instructions

1. Run `composer install`
2. Run `vendor/bin/psalm`
3. It is expected to get thousands of issues.

```
------------------------------
5385 errors found
------------------------------
8789 other issues found.
You can display them with --show-info=true
------------------------------
```


## Pre-Merge Checklist
